### ISSUE TITLE
why we use 800 as the default hiddensize

### ISSUE THREAD

**Issue by xhzhao** [SeanNaren/deepspeech.pytorch#233]
we found that the DS2 paper uses many hidden sizes such as 1760,2560, ... and some [paper](https://arxiv.org/abs/1704.05119) use 1760 too.
My question is:
1. Why we use 800 as default HiddenSize?
    From my understanding, the 1760 model will be about 4x larger than the default 800 model.
2. Did you guys tried on the 1760 HiddenSize? Any result about that?

**Comment by ryanleary** [comment]
800 GRU cells offered a good tradeoff for training speed and accuracy by virtue of limiting the total number of free parameters for purposes of providing pretrained models. We encourage you to experiment with alternative architectures. We would also encourage you to contribute pretrained models based on alternative architectures.
